### DISCUSSION TITLE
Whatsapp section does not support multi-device Whatsapp web login?

### DISCUSSION THREAD

**Discussion by rahmatullahbarkat** [42wim/matterbridge#1741] [Help / Q&A]
I tried to login by scanning the QR Code from WhatsApp that already joined the Multi-device beta program, then it showed error panic: interface conversion: interface {} is nil, not map[string]interface {} 

goroutine 1 [running]:
github.com/Rhymen/go-whatsapp.newInfoFromReq(0xc00060c8c0)
        /go/src/github.com/42wim/matterbridge/vendor/github.com/Rhymen/go-whatsapp/session.go:63 +0x951
github.com/Rhymen/go-whatsapp.(*Conn).Login(0xc000862000, 0xc000852000)
        /go/src/github.com/42wim/matterbridge/vendor/github.com/Rhymen/go-whatsapp/session.go:268 +0xd36
github.com/42wim/matterbridge/bridge/whatsapp.(*Bwhatsapp).Login(0xc00009f110)
        /go/src/github.com/42wim/matterbridge/bridge/whatsapp/whatsapp.go:138 +0x10f
github.com/42wim/matterbridge/bridge/whatsapp.(*Bwhatsapp).Connect(0xc00009f110)
        /go/src/github.com/42wim/matterbridge/bridge/whatsapp/whatsapp.go:83 +0x2c5
github.com/42wim/matterbridge/gateway.(*Router).Start(0xc00055e000)
        /go/src/github.com/42wim/matterbridge/gateway/router.go:76 +0x35f
main.main()
        /go/src/github.com/42wim/matterbridge/matterbridge.go:65 +0x3be

Is it because of the Whatsapp only still supported the old way of logging in? Is there any solution? I use Windows 64bit

**Comment by 42wim** [comment]
Try the latest v1.25.0 release and read the changelog/readme

**Comment by junior448** [comment]
mine has this same error, after I installed another version of whatsapp, now I'm not getting it anymore

**Comment by ghost** [comment]
Any solution?
